webpack/client/webpack.prod.config.js

```diff
@@ -22,6 +24,10 @@ const plugins = [
     __DEVELOPMENT__: false,
     __DEVTOOLS__: false, // For disabling redux devtools on Production mode
   }),
+  new InjectManifest({
```

Comment:
@shubhamsWEB - Can you please mention the details what is the significance of InjectManifest here?

Reply:
The Injectmanifest plugin injects a list of URLs and revision information for precaching based on the webpack asset pipeline. And add them in a service-worker.js file in the dist.